pkg/nodejs/yarn/parse_testcase.go

```diff
@@ -3,19 +3,23 @@ package yarn
 import "github.com/aquasecurity/go-dep-parser/pkg/types"
 
 var (
-	// docker run --name node --rm -it node:12-alpine sh
+	// cd ./pkg/nodejs/yarn
+	// docker build -t yarn-test testcase_deps_generator
+	// docker run --name node --rm -it yarn-test sh
 	// yarn init -y
 	// yarn add promise jquery
 	// yarn list | grep -E -o "\S+@[^\^~]\S+" | awk -F@ 'NR>0 {printf("{\""$1"\", \""$2"\", \"\"},\n")}'
 	// to get deps with locations from lock file use following commands:
-	// awk '/^\S+@[~^*]?(>= )?[0-9.]*/,/^$/{if($0=="") {print "--"prev} else { if(substr($0,1,2)!="  ") {print NR":"$0} else {print $0}} prev=NR}; END{print "--"prev}' | awk 'BEGIN {s=""}; {(substr($0,1,2)=="--") ? (s=s$0"\n") : (s=s$0)}; END { print s}' | sed -E 's/@([0-9~><*\^]|npm).*version:? "?/:/' | sed 's/  /:/' | sed 's/"//g'| awk 'match($0, /[[:digit:]]+$/) {print substr($0, RSTART, RLENGTH)":"$0 }' |  awk -F":" '{print "{Name: \""$3"\", Version: \""$4"\", Locations: []types.Location{{StartLine: "$2", EndLine: "$1"}}},"}'
+	// cat yarn.lock | awk '/^\S+@[~^*]?(>= )?[0-9.]*/,/^$/{if($0=="") {print "--"prev} else { if(substr($0,1,2)!="  ") {print NR":"$0} else {print $0}} prev=NR}; END{print "--"prev}' | awk 'BEGIN {s=""}; {(substr($0,1,2)=="--") ? (s=s$0"\n") : (s=s$0)}; END { print s}' | sed -E 's/@([0-9~><*\^]|npm).*version:? "?/:/' | sed 's/  /:/' | sed 's/"//g'| awk 'match($0, /[[:digit:]]+$/) {print substr($0, RSTART, RLENGTH)":"$0 }' |  awk -F":" '{print "{Name: \""$3"\", Version: \""$4"\", Locations: []types.Location{{StartLine: "$2", EndLine: "$1"}}},"}'
 	yarnNormal = []types.Library{
 		{Name: "asap", Version: "2.0.6", Locations: []types.Location{{StartLine: 5, EndLine: 8}}},
 		{Name: "jquery", Version: "3.4.1", Locations: []types.Location{{StartLine: 10, EndLine: 13}}},
 		{Name: "promise", Version: "8.0.3", Locations: []types.Location{{StartLine: 15, EndLine: 20}}},
 	}
 
-	// yarn list --json --no-progress | jq -r '[ .data.trees[] | { ID: .name, DependsOn: [ .children[] | { name: .name, color: .color } ] } ]'
+	// ... and
+	// yarn --cwd test_deps_generator install
```

Comment:
i get error in this command:
```sh
/ # yarn --cwd test_deps_generator install
Error: ENOTDIR: not a directory, open 'test_deps_generator/.yarnrc'
    at Object.openSync (fs.js:498:3)
    at readFileSync (fs.js:394:35)
    at /opt/yarn-v1.22.19/lib/cli.js:97395:58
    at Array.map (<anonymous>)
    at parseRcPaths (/opt/yarn-v1.22.19/lib/cli.js:97393:78)
    at Object.findRc (/opt/yarn-v1.22.19/lib/cli.js:97407:10)
    at getRcConfigForCwd (/opt/yarn-v1.22.19/lib/cli.js:56965:74)
    at buildRcArgs (/opt/yarn-v1.22.19/lib/cli.js:57011:16)
    at getRcArgs (/opt/yarn-v1.22.19/lib/cli.js:57067:16)
    at /opt/yarn-v1.22.19/lib/cli.js:88330:76

```
Can you take a look?

Reply:
Sorry, fixed the dockerfile

Reply:
looks like it works good!